.travis.yml

```diff
@@ -29,26 +29,11 @@ android:
     # Latest artifacts in local repository
     - extra-google-m2repository
     - extra-android-m2repository
-    - android-sdk-license-.+
-    - '.+'
     # Specify at least one system image
-    - sys-img-armeabi-v7a-google_apis-$ANDROID_API_LEVEL
-    - sys-img-armeabi-v7a-android-$ANDROID_API_LEVEL_22
-
-# list of directories to Cache
-
-before_install:
-  - mkdir "$ANDROID_HOME/licenses" || true
-  - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
-  - echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd" > "$ANDROID_HOME/licenses/android-sdk-preview-license"
-
-licenses:
```

Comment:
`licenses` used to [work on Sep16](https://github.com/travis-ci/travis-ci/issues/6555#issuecomment-244499595) but stopped working [by Nov16](https://github.com/travis-ci/travis-ci/issues/6555#issuecomment-258624113)

Reply:
I wonder if this is just ignored by travis